apis/Google.Cloud.Diagnostics.AspNet/Google.Cloud.Diagnostics.AspNet/ErrorReporting/IWebApiExceptionLogger.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright 2018 Google Inc. All Rights Reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Web.Http.Controllers;
+
+namespace Google.Cloud.Diagnostics.AspNet
+{
+    /// <summary>
+    /// A generic exception logger for use from withing a Web Api application.
```

Comment:
withing => within
Api => API

I'd probably remove "generic" as well, in that it's not generic in C# terms.

Reply:
Yes, agreed. Done.

Reply:
Also expanded on the comment of IExceptionLogger saying that is used from within a Web API controller HTTP context info might not be logged.